Dockerfile

```diff
@@ -422,14 +422,18 @@ ENV GIMME_GO_VERSION "1.12"
 ENV GIMME_ENV_PREFIX "/opt/buildhome/.gimme/env"
 RUN gimme
 
-
+################################################################################
+#
+# Wasmer
+#
+################################################################################
+RUN curl https://get.wasmer.io -sSfL | sh
 
 ################################################################################
 #
 # Dotnet Core
 #
 ################################################################################
-USER buildbot
```

Comment:
Is this a mistake?

Reply:
no, it's not necessary. The current user at that point is already buildbot, so this instruction was adding an extra layer without any reason.